client/app/bundles/shared/components/Resource.scss

```diff
@@ -0,0 +1,38 @@
+@import './base';
+
+.container {
+  position: relative;
+  width: 100%;
+  font-family: Lato;
+  padding: 25px 10px;
+
+  &:before {
+    content: '';
+    position: absolute;
+    left: 10px;
+    top: 9px;
+    border: 1px solid #fff;
+    border-width: 0 40px 0 0;
+    height: 3px;
+  }
+  
+  .link {
+    font-size: $font24;
+    color: $background-color;
+    padding-bottom: 10px;
+  }
+  
+  div > a {
+    text-decoration: none;
+    color: $header-text-color;
+  }
+
+  .author {
+    color: $author-text-color;
```

Comment:
The author text color should be 50% opacity. Maybe use the **rgba** sass function.